.gitattributes

```diff
@@ -0,0 +1,17 @@
+# Source code
+*.kt            text eol=lf
```

Comment:
Make the default LF and only add exemptions. Like https://github.com/square/retrofit/blob/master/.gitattributes. Less whack-a-mole.